calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
```

Comment:
Do you think it would be useful to have the variable name indicate the data type here so it's clear the grades are coming in an array? Something like `gradeArray `perhaps.